src/main/java/seedu/duke/projects/Project.java

```diff
@@ -0,0 +1,14 @@
+package seedu.duke.projects;
+
+public class Project {
+    private String title;
+
```

Comment:
Maybe we need to add description to a project?

Reply:
Good point; I will add it to the issues tracker